src/ff/galois_field.rs

```diff
@@ -481,3 +482,12 @@ bit_array_impl!(
     // x^8 + x^4 + x^3 + x + 1
     0b1_0001_1011_u128
 );
+
+bit_array_impl!(
+    bit_array_1,
+    Gf2,
```

Comment:
A binary field. Just two elements, 0 and 1.

Reply:
Is this Fp2 or GF(2)?

Reply:
I think those two are isomorphic, so I guess it's both. But given the way that multiplication is defined here, it sort of looks and smells more like a Galois Field.